sources/code/main.ts

```diff
@@ -96,7 +96,11 @@ const chromiumVersion: string = process.versions.chrome;
  * if you're improving the code of this application
  */
 
-let appContributors: Array<string> = [appAuthor];
+ //console.log(packageJson)
+ //console.log(packageJson.contributors)
+
+//let appContributors: Array<string> = [appAuthor + packageJson.contrib.authors];
+let appContributors: Array<string> = [ appAuthor, ...packageJson.contributors ];
```

Comment:
Why you have changed that? `appAuthor` is added there as a fallback value, other `if` functions overwrite this array if neccessary...